### DISCUSSION TITLE
Cannot find module "#components" since rc-8

### DISCUSSION THREAD

**Discussion by davidfrgla** [nuxt/nuxt#17070] [Questions] [answered]
Hi, I upgraded nuxt to rc-8 and on files that direct-import components I have this warn:
`Cannot find module '#components' or its corresponding type declarations.`
Any clue ?

**Comment by danielroe** [comment, marked as answer]
Type support should be resolved in edge: https://github.com/nuxt/framework/pull/6634

For now, you can import from `#build/components` with type support.

**Reply by davidfrgla** [reply]
thanks !
